pallets/parachain-staking/src/set.rs

```diff
@@ -16,10 +16,10 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
-use frame_support::{traits::Get, BoundedVec, DefaultNoBound, RuntimeDebug};
+use frame_support::{traits::Get, BoundedVec, DefaultNoBound};
 use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
-use sp_runtime::{traits::Zero, SaturatedConversion};
+use sp_runtime::{traits::Zero, SaturatedConversion, RuntimeDebug};
```

Comment:
Ooh, `RuntimeDebug` can be accessed via `sp_runtime`.

So we can actually:
1. Remove `sp-debug-derive` dependency in runtime/integration-tests/Cargo.toml;
2. Then replace `use sp_debug_derive::RuntimeDebug;` in runtime/integration-tests/src/sibling.rs with 
`use sp_runtime::RuntimeDebug`

Reply:
Alright! Let me try.